Documentation/PackageDescription.md

```diff
@@ -306,6 +306,16 @@ static func package(url: String, _ requirement: Package.Dependency.Requirement)
 ///     - branch: A dependency requirement. See static methods on `Package.Dependency.Requirement` for available options.
 static func package(name: String? = nil, url: String, branch: String) -> Package.Dependency
 
+/// Adds a remote package dependency given a branch requirement.
```

Comment:
Should this say revision?

Reply:
Yeah you're right, thanks for catching that